src/utils/utils_log.c

```diff
@@ -32,6 +32,29 @@
 #include "utils_common.h"
 #include "utils_log.h"
 
+#define UMF_VERSION_MAGIC_STR "\x00@(#) "
+#define UMF_VERSION_PREF_STR "Intel(R) "
```

Comment:
is this some requirement to have this prefix...?

Reply:
No.